<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2I3TPA0" ignored="4" name="Create Portfolio Test" tests="2" failures="2" timestamp="15 Oct 2022 03:38:50 GMT" time="29.81" errors="0">
  <testcase name="@AfterTest quit" time="0.0" classname="testbase.BaseTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Keywords.GenericKeywords.quit(GenericKeywords.java:292)
at testbase.BaseTest.quit(BaseTest.java:62)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest quit -->
  <testcase name="@AfterTest quit" time="0.0" classname="testbase.BaseTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Keywords.GenericKeywords.quit(GenericKeywords.java:292)
at testbase.BaseTest.quit(BaseTest.java:62)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest quit -->
  <testcase name="doLogin" time="29.44" classname="testcases.rediff.Session">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Keywords.GenericKeywords.navigate(GenericKeywords.java:97)
at testcases.rediff.Session.doLogin(Session.java:19)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="createPortfolio" time="0.001" classname="testcases.rediff.PortfolioManagement">
    <failure type="java.lang.IllegalArgumentException" message="Input must be set">
      <![CDATA[java.lang.IllegalArgumentException: Input must be set
at org.openqa.selenium.internal.Require.nonNull(Require.java:51)
at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:100)
at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:128)
at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:61)
at Keywords.GenericKeywords.isElementPresent(GenericKeywords.java:125)
at Keywords.GenericKeywords.getWebElement(GenericKeywords.java:111)
at Keywords.GenericKeywords.click(GenericKeywords.java:107)
at testcases.rediff.PortfolioManagement.createPortfolio(PortfolioManagement.java:16)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- createPortfolio -->
  <testcase name="doLogout" time="0.0" classname="testcases.rediff.Session">
    <ignored/>
  </testcase> <!-- doLogout -->
  <testcase name="@BeforeTest beforeTest" time="0.0" classname="testbase.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase name="deletePortfolio" time="0.0" classname="testcases.rediff.PortfolioManagement">
    <ignored/>
  </testcase> <!-- deletePortfolio -->
  <testcase name="@AfterTest quit" time="0.0" classname="testbase.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterTest quit -->
</testsuite> <!-- Create Portfolio Test -->
